src/types/domain/Commander.ts

```diff
@@ -6,5 +6,6 @@ export type Commander = {
      * A collection of match IDs of matches that the commander is played in
      */
     matches: string[];
+    validMatches: number; // Games we want to take into account when calculating stats (usually 4 player pod)
```

Comment:
same comments here as what was stated in Player.ts

Reply:
Fixed.